src/app/+dashboard/components/greeting/greeting.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="container-greet">
+  <h1><span>{{ dayTime() }}</span>Valia!</h1>
```

Comment:
It is not allowed to call function from the template for data displaying (you call function to handle some events, for example 'click')
For more information you can read [here](https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496)

Reply:
fixed